manifests/monitoring/thanosConfig.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if not (index .thanos "disabled") }}
```

Comment:
is this if statement required?

Reply:
Not required, but without it will have Thanos configmap even if we don't have Thanos itself.

Reply:
Is the `ApplySpec('monitoring/thano-config/yaml')` not guarded by an If statement in install.go ?